.eslintignore

```diff
@@ -0,0 +1 @@
+addon/ng2/blueprints/ng2/files/src/config/system.config.js
```

Comment:
Maybe using `eval` instead of `JSON.parse` (in `loadSystemJson()`) would be a viable alternative and remove the need for double quotes.

I don't think there are any performance or security concerns here, but still `eval` is ugly :confused: